packages/kbn-config/src/raw/read_config.test.ts

```diff
@@ -50,13 +62,13 @@ test('should throw an exception when referenced environment variable in a config
 test('throws parsing a config with forbidden paths', () => {
   expect(() =>
     getConfigFromFiles([fixtureFile('forbidden_1.yml')])
-  ).toThrowErrorMatchingInlineSnapshot(`"Forbidden path detected: test.aaa.__proto__.hello"`);
+  ).toThrowErrorMatchingInlineSnapshot(`"Forbidden path detected: test.aaa['__proto__.hello']"`);
```

Comment:
(The behavior did not change, only the way we format the forbidden path due to the implementation changing)